base/docs/helpdb/Base.jl

```diff
@@ -784,12 +784,12 @@ Equivalent to `stat(file).mode`
 filemode
 
 """
-    print_joined(io, items, delim, [last])
+    join(io, items, delim, [last])
```

Comment:
Yes I mean the part outside the docstring. Is it ok for the signature to have the brackets, but the part outside not to?